src/CMakeLists.txt

```
@@ -18,6 +18,11 @@ set (src_zlib_DIST
    src/zlib-1.2.13/zlib.h
    src/zlib-1.2.13/zutil.h
 )
+
+set (src_utf8proc_DIST
+   src/utf8proc-2.8.0/utf8proc.h
+   src/utf8proc-2.8.0/utf8proc.c
```

```suggestion
   src/utf8proc-2.8.0/utf8proc.c
   src/utf8proc-2.8.0/utf8proc_data.c
```

This may resolve the error in the [make_release_archive](https://parsley.mongodb.com/evergreen/mongo_c_driver_releng_make_release_archive_patch_7d5beeae25e3c821b6e01e204ba6d323184c8b7d_64998425306615e8c37fede6_23_06_26_12_27_19/0/task?bookmarks=0,531&shareLine=385):

```
/Users/kevin.albertson/review/mongo-c-driver-1319/cmake-build/mongo-c-driver-1.24.0-pre/src/utf8proc-2.8.0/utf8proc.c:53:10: fatal error: 'utf8proc_data.c' file not found
#include "utf8proc_data.c"
         ^~~~~~~~~~~~~~~~~
1 error generated.
```

The utf8proc_data.c file needs to be included in the distribution tarball.

This error can be reproduced locally by running the `distcheck` target:
```
cmake --build cmake-build --target distcheck -- -j16
```
The `distcheck` target builds the distribution tarball, extracts it, then rebuilds the driver.